ports/mecab/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "mecab",
   "version-date": "2019-09-25",
-  "port-version": 4,
+  "port-version": 5,
   "description": "A morphological analysis engine based on CRF",
   "supports": "!(uwp | arm | arm64)",
```

Btw it supports arm on macOS. So:
```suggestion
  "supports": "!uwp & !(arm & windows)",
```